TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/support/Room.java

```diff
@@ -0,0 +1,17 @@
+package com.novoda.tpbot.support;
+
+public enum Room {
```

Comment:
I want to remove this soon with another call to the server to fetch a collection of `Rooms` that the app can use. That way, if we want to add rooms later we don't need to redeploy the app.